### ISSUE TITLE
Plot move java.lang.NullPointerException error

### ISSUE THREAD

**Issue by x1p** [IntellectualSites/PlotSquared#2269]
# Bug report template 

**Debug paste link:** 

https://incendo.org/paste/view/976ddbdcc7964710aae6a02c4af1edda

**Description of the problem:** 
Plot move turns into java.lang.NullPointerException
https://pastebin.com/hM1NBdQ9

**How to replicate:** 
PlotSquared-19.2.8-17027e6-152
Plot move
Error

**Checklist**:

- [X] I included a `/plot debugpaste` link
- [X] I made sure there are no duplicates of this report [(Use Search)](https://github.com/IntellectualSites/PlotSquared/issues?utf8=%E2%9C%93&q=is%3Aissue)
- [X] I made sure I am using an up-to-date version of PlotSquared
- [X] I made sure the bug/error is not caused by any other plugin

**Comment by MattBDev** [comment]
Can you attempt to recreate this issue with a build from February 11th? Just send the error with pastebin if it comes up.

**Comment by x1p** [comment]
Nope.
```
[16:19:51 WARN]: java.lang.NullPointerException
[16:19:51 WARN]: at com.github.intellectualsites.plotsquared.plot.database.SQLManager.sendBatch(SQLManager.java:354)
[16:19:51 WARN]: at com.github.intellectualsites.plotsquared.plot.database.SQLManager.lambda$new$0(SQLManager.java:141)
[16:19:51 WARN]: at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftTask.run(CraftTask.java:82)
[16:19:51 WARN]: at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:52)
[16:19:51 WARN]: at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
[16:19:51 WARN]: at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[16:19:51 WARN]: at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[16:19:51 WARN]: at java.lang.Thread.run(Thread.java:748)
```

**Comment by dordsor21** [comment]
Ought to be fixed in 3e8308e though there shouldn't have been any issues arising from the NPE, since the following method simply nulled the variable anyway.
